test/index.spec.js

```diff
@@ -80,22 +80,22 @@ describe('index.js', () => {
 
     expect(
       () => migrator.getLatestMigrationVersion([])
-    ).to.throw('Migrations must be non-empty array!')
+    ).to.throw(errors.InvalidValueError, 'Migrations must be non-empty array!').with.property('code', errors.InvalidValueError.code)
```

Comment:
Let's just test the code field (not the text of the error message itself) so that we can easily change the text later without having to change the tests

Reply:
Yeah that is good idea, will fix this.